site/styles/_forms.scss

```diff
@@ -88,20 +91,40 @@ s72-activatedevice-form,
   .s72-datepicker-year {
     flex-shrink: 0;
   }
+  .s72-datepicker-month {
+    width: 100%;
+    margin: 0 4px;
+  }
 }
 // Form Control Default
 .s72-form-control {
   font-weight: $font-weight-normal;
+  height: var(--input-height);
+
+  &:not(#new-card-container) {
+    background-color: $body-bg-accent;
+    border-color: rgba($color: var(--body-color-rgb), $alpha: 0.1);
```

Comment:
Could you change this (and the other instances of it) to `rgba(var(--body-color-rgb), 0.1);`.  I dislike using the `$` keyword arguments in non-Sass functions.

Reply:
Done ✅